test/FunctionalTests/Microsoft.OData.Core.Tests/ScenarioTests/Streaming/ODataJsonLightStreamReadingTests.cs

```diff
@@ -0,0 +1,1312 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="ODataJsonLightStreamReadingTests.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using FluentAssertions;
+using Microsoft.OData.Edm;
+using Microsoft.OData.JsonLight;
+using Microsoft.Test.OData.DependencyInjection;
+using Xunit;
+using System.Threading.Tasks;

```

Comment:
It seems you should order by using.

Reply:
Fixed here and in ODataJsonLightStreamWritingTests

---
In reply to: [268351497](https://github.com/OData/odata.net/pull/1204#discussion_r268351497) [](ancestors = 268351497)